mergebot_backend/merge.py

```diff
@@ -260,6 +279,17 @@ def flush_queue(self):
         """
         pr = self.work_queue.get_nowait()
         self.publisher.publish_dequeue(item_id=pr.get_num())
+        pr.post_commit_status(
+            state=github_helper.COMMIT_STATE_ERROR,
+            url=MERGEBOT_ITEM_URL.format(
+                name=self.config.name, number=pr.get_num()),
+            description='MergeBot shutdown; resubmit when mergebot is back up.',
+            context='MergeBot: Merge',
+            logger=self.merge_logger)
+        # In the case of shutdown, we still post back to the pull request
+        # because otherwise if mergebot is stopped and restarted we will try
+        # this pull request again. If mergebot is modified to work with webhooks
+        #  this can be removed.
         pr.post_info('MergeBot shutting down; please resubmit when MergeBot is '
```

Comment:
I'm not sure what you mean by this -- in this section we're both updating the commit status (line ~286), and posting a comment on the PR indicating the same message.